dtfs-central-api/src/v1/controllers/utilisation-report-service/put-utilisation-report-status.controller.ts

```diff
@@ -0,0 +1,97 @@
+import { Request, Response } from 'express';
+import { MongoClient, ObjectId } from 'mongodb';
+import db from '../../../drivers/db-client';
+import {
+  ReportDetails,
+  ReportFilter,
+  ReportId,
+  UtilisationReportReconciliationStatus,
+  ReportWithStatus,
+  UpdateUtilisationReportStatusInstructions,
+} from '../../../types/utilisation-reports';
+import { UploadedByUserDetails } from '../../../types/db-models/utilisation-reports';
+import { TfmUser } from '../../../types/db-models/tfm-user';
+import { updateManyUtilisationReportStatuses } from '../../../services/repositories/utilisation-reports-repo';
+
+type RequestBody = {
+  user: TfmUser;
+  reportsWithStatus: ReportWithStatus[];
+};
+
+class PayloadError extends Error {
+  status: number;
+
+  constructor(message: string) {
+    super(message);
+    this.status = 400;
+  }
+}
```

Comment:
Nice use of a custom error here : )

I think we may also need to add
```
this.name = this.constructor.name;
```
in the constructor (see other examples in Portal API).

It might also be nice to make this shared so we can use this in other places where there are payload issues. If so I would put this in `src/v1/errors` (to match what we've done in other projects like Portal API).